src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -2910,11 +2925,27 @@ class ArnoldGlobals
 		{
 			updateCameraMeshes();
 
+			AtNode *options = AiUniverseGetOptions();
+
 			AiNodeSetInt(
-				AiUniverseGetOptions(), g_aaSeedArnoldString,
+				options, g_aaSeedArnoldString,
 				m_aaSeed.get_value_or( m_frame.get_value_or( 1 ) )
 			);
 
+			AtNode *dicingCamera = nullptr;
+			if( m_subdivDicingCameraName.size() )
+			{
+				dicingCamera = AiNodeLookUpByName( AtString( m_subdivDicingCameraName.c_str() ) );
+			}
```

Comment:
Let's issue an `IECore::msg::warning()` here if we can't find the camera, so users aren't left in the dark.